Core/UIDeviceHelper/UIDevice+ModelDescription.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  UIDevice+ModelDescription.swift
+//  CLArena
+//
+//  Created by Ross Patman on 12/02/2018.
+//  Copyright © 2018 ConcertLive. All rights reserved.
+//
+
+import UIKit
+
+extension UIDevice {
```

Comment:
Is this code also in Frontier, and does it need to be removed if so?

Reply:
@rossp-av  Yes, I'll remove it from Frontier once I import this SDK version. Will update in the same PR: https://bitbucket.org/livestyled-dev/frontier.ios/pull-requests/2675